go/cmd/vtctldclient/command/schema.go

```diff
@@ -109,7 +109,7 @@ func commandApplySchema(cmd *cobra.Command, args []string) error {
 	var allSQL string
 	if applySchemaOptions.SQLFile != "" {
 		if len(applySchemaOptions.SQL) != 0 {
-			return errors.New("Exactly one of --sql and --sql-file must be specified, not both.")
+			return errors.New("exactly one of --sql and --sql-file must be specified, not both")
```

Comment:
So I disagree with the linter here. This error text gets output directly to the user, so to me it's better to be a full sentence with capitalization and punctuation.

Reply:
added `// nolint`